src/scenes/home/faq/question/question.js

```diff
@@ -8,6 +8,7 @@ class Question extends Component {
       <div className={styles.accordionSingle}>
         <input className={styles.accordionSingleHidden} type="checkbox" id={this.props.q} />
         <label className={styles.accordionSingleQuestion} htmlFor={this.props.q}> {this.props.q}</label>
+        { // eslint-disable-next-line react/no-danger react/prop-types: 0) }
```

Comment:
We're getting: "Syntax error: Adjacent JSX elements must be wrapped in an enclosing tag (13:6)" from Travis.

Try doing `{/* eslint-disable-next-line react/no-danger react/prop-types: 0 */}`

I can't dive in myself on this one cuz I'm at work - apologies.